aws/resource_aws_iam_user.go

```diff
@@ -121,6 +127,7 @@ func resourceAwsIamUserRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("permissions_boundary", output.User.PermissionsBoundary.PermissionsBoundaryArn)
 	}
 	d.Set("unique_id", output.User.UserId)
+	d.Set("tags", tagsToMapIAM(output.User.Tags))
```

Comment:
We should perform error checking here to catch any unexpected type conversion issues. 👍 

```suggestion
	if err := d.Set("tags", tagsToMapIAM(output.User.Tags)); err != nil {
		return fmt.Errorf("error setting tags: %s", err)
	}
```